website/content/terminal/usage/basics/index.md

```diff
@@ -1,114 +1,280 @@
+![The Home Screen](https://user-images.githubusercontent.com/85772166/233247655-2f8d0dae-be68-48ca-9b35-123b5b985cb6.png)
 
-Throughout the entire terminal, the same set of colors are used which all share the same representation. This is structured as follows:
+## Auto Complete
 
-- <b><span style={{color:"#00AAFF"}}>Light Blue</span></b>: represents commands.
-- <b><span style={{color:"#005CA9"}}>Dark Blue</span></b>: represents menus, also recognizable by the `>` in front.
-- <b><span style={{color:"#EF7D00"}}>Orange</span></b>: represents titles and headers.
-- <b><span style={{color:"#FCED00"}}>Yellow</span></b>: represents descriptions of a parameter or variable.
-- <b>White</b>: represents text, usually in combination with a description that is in Yellow.
+The OpenBB Terminal is equipped with an auto completion engine that presents choices based on the current menu.  Whenever you start typing, the prompt will appear. When the function contains optional arguments, pressing the `space bar` after typing the command will present the list of available arguments.  This functionality dramatically reduces the number of key strokes required to perform tasks and, in many cases, eliminates the need to consult the help dialogue for reminders.  Choices - where they are bound by a defined list - are searchable with the up and down arrow keys.
 
-### Explanation of Menus
+![Auto Complete](https://user-images.githubusercontent.com/85772166/233247702-f707531c-2c65-4380-a662-cd4bc2ae0199.png)
 
-Menus, depicted in <b><span style={{color:"#005CA9"}}>Dark Blue</span></b>, take you to a new section of the terminal referred to as a menu. For example, if you wish to view information about stocks, you can do so by typing `stocks` and pressing `ENTER` (⏎). This opens a new menu as depicted below.
+## Help Dialogues
 
-<img width="800" alt="Explanation of Menus" src="https://user-images.githubusercontent.com/46355364/225117373-93787a40-cb26-4599-86b7-7bd023b31e4b.png"></img>
+### -h or --help
 
-Depending on the menu you are in, you are presented with a new set of commands and menus you can select. There are interactions in place between each menu. For example, when selecting a company within the `stocks` menu, the terminal will remember your selection when you visit the `fa` or `options` menu. See [Introduction to Stocks](/terminal/usage/intros/stocks).
+A help dialogue for any function at the current location is printed to the screen by attaching `-h` to the command.  The information returned contains a short description of the function and all accepted arguments.  For an example, the `/news` function:
 
-:::note **About Pipelines**
-You can quickly jump between menus by using a forward slash (`/`). For example, if I want to access the options menu, You can type `/stocks/options` to instantly arrive at this menu. You can do this from any location within the OpenBB Terminal!
-:::
```

Comment:
I think this should definitely still be incorporated here.

Reply:
This is included near the top:
![Screenshot 2023-04-21 at 11 57 01 AM](https://user-images.githubusercontent.com/85772166/233714098-ed4dc90a-e17e-434b-95cc-c6b581324fbd.png)

Reply:
I like on the website how it is visualized as a special box.  Maybe we can keep the box and start it from the sentence you have as "Absolute path are also valid" ?